.github/workflows/lint-migrations.yml

```
@@ -0,0 +1,73 @@
+name: Eugene CI check
+
+env:
+  EUGENE_VERSION: "0.6.1"
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  lint-migration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v2
```

```suggestion
        uses: actions/setup-python@v5
```

Is the latest version